### ISSUE TITLE
The maximum length of a fixed length record may be incorrectly defaulted

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#5]
```
If minLength is set on a fixed length record and maxLength is not set, 
maxLength will incorrectly default to minLength which may be less than the sum 
of all defined field lengths.

Affects release 0.9.2.  

Workaround: when a minLength value is configured on a fixed length record, also 
set maxLength.
```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 2 Feb 2011 at 3:38

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 2 Feb 2011 at 7:38
- Changed title: **The maximum length of a fixed length record may be incorrectly defaulted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 0.9.3.
```

Original comment by `kevin.s...@gmail.com` on 8 Feb 2011 at 2:50
- Changed state: **Fixed**
- Added labels: **Milestone-0.9.3**
